pokerole_irl/worlds_app/urls.py

```diff
@@ -0,0 +1,11 @@
+from django.urls import path
+
+from .views import SettingListView, SettingCreateView, SettingView, CharacterView, CharacterCreateView
+
+urlpatterns = [
+    path('setting_list/', SettingListView.as_view(), name='setting-list'),
```

Comment:
I think a better URL scheme here would just be "list/", since we are already on the "worlds/" url.

Reply:
Renamed